templates/app/client/components/auth(auth)/router.decorator.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-import _ from 'lodash';
+import * as _ from 'lodash';
```

Comment:
How does this change anything? Shouldn't the same methods be available on both the default export and the named exports?